packages/nimble-components/src/rich-text-mention/users/view/template.ts

```diff
@@ -5,6 +5,8 @@ import type { RichTextMentionUsersView } from '.';
 // that arises when the mention user view is activated at the end of a line in the rich text editor while in edit mode.
 // This can be removed when the below issue is resolved
 // See: https://github.com/ni/nimble/issues/1659
-export const template = html<RichTextMentionUsersView>`<span class="control"
+export const template = html<RichTextMentionUsersView>`<span
+        class="control"
+        part="control"
```

Comment:
For disabling view element when editor get disabled, query selecting all the mention view node and setting the property `disabled` to true actually made it to disabled state. Still, we tried finding some other ways to update the `disabled` dynamically but didn't get any. So, we just modified it in CSS by accessing it through part in editor styles. We also went through the [comments ](https://github.com/ni/nimble/pull/1653#discussion_r1386960217)that Milan mentioned not to expose the `part` unnecessarily.

Please let us know your thoughts on this, shall we go with the `css` approach or through `queryselecting` one?

Reply:
I have updated the suggested approach by keeping new CSS variable and toggling it on editor disabling. But anyway, keeping this open to know all your thoughts on this.